lib/module.js

```diff
@@ -163,24 +163,19 @@ export default function SentryModule (moduleOptions) {
   }
 
   // Enable publishing of sourcemaps
-  if (!options.disabled) {
-    this.extendBuild((config, { isClient, isModern, isDev }) => {
-      if (!options.publishRelease || isDev) {
-        return
-      }
+  if (options.publishRelease && !options.disabled && !this.options.dev) {
+    this.nuxt.hook('webpack:config', (webpackConfigs) => {
```

Comment:
I've just check and this hook was introduced in [v2.10.0](https://github.com/nuxt/nuxt.js/pull/6468).
That's pretty recent. I'm not sure what to do about that. It might require major version change and note in breaking changes...

Reply:
Yeah, I'll have to go with major release.